### DISCUSSION TITLE
Business owners

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#393] [General]
What is the best business advice did you ever receive from someone?
Please share the wisdom. 
That is not only curiosity, but I am a new business owner and looking forward to some good advices.

**Comment by hernessteve** [comment]
Build a friendly team, learn to delegate jobs as you are only one person and can't do everything yourself (that was a very difficult thing for me to do). If it comes to promotions and marketing  you can always [send sms from slack ](https://clerk.chat/slack)and inform your customers about everything what is important for you and your business.  Sometimes you can just send them just a little reminder about yourself and believe me some will come to you in a short term. This thing actually works.
